build/build-options.js

```diff
@@ -7,14 +7,20 @@ const {
   FILES,
   FOLDERSEARCH,
   NO_PROGRESSBAR,
+  FIX_FLAWS,
+  FIX_FLAWS_DRY_RUN,
+  FIX_FLAWS_VERBOSE,
 } = require("./constants");
 
-const options = {
+const options = Object.freeze({
```

Comment:
This was a change I made last week. I don't even remember why. Just felt right at the time. 

However, I can still see a future where you start your server with `BUILD_FIX_FLAWS=false` for example, but through some UI you press a button which triggers another run of `buildDocument(document)` but this time you want to temporarily change it so that `options.fixFlaws === true` this *one* run. 

Let's worry about it when we get there. It still feels pretty solid to not allow anything to change the module level mutable globals.